csrc/quantization/gptq_marlin/gptq_marlin.cu

```diff
@@ -537,6 +537,7 @@ __global__ void Marlin(
     int prob_m,           // batch dimension m
     int prob_n,           // output dimension n
     int prob_k,           // reduction dimension k
+    int stride_am,        // A.stride(0), equal to prob_k is A is contiguous
```

Comment:
light suggestion of naming this `lda` - stands for the "leading dimension of A", and is the name used in [GEMM](https://www.netlib.org/lapack/explore-html/dd/d09/group__gemm_ga1e899f8453bcbfde78e91a86a2dab984.html#ga1e899f8453bcbfde78e91a86a2dab984) for this concept, and fits in with our usage here:

https://github.com/vllm-project/vllm/blob/fd8e055ffba508e094cd1793e49bbdc5e53b7266/csrc/quantization/cutlass_w8a8/scaled_mm_c2x.cuh#L144-L146